spec/controllers/home_controller_spec.rb

```diff
@@ -4,9 +4,60 @@
   render_views
 
   describe 'GET #index' do
-    it 'redirects to about page' do
-      get :index
-      expect(response).to redirect_to(about_path)
+    context 'when not signed in' do
+      it 'redirects to about page' do
+        get :index
+        expect(response).to redirect_to(about_path)
+      end
+    end
+
+    context 'when signed in' do
+      let(:user) { Fabricate(:user) }
+      subject do
+        sign_in(user)
+        get :index
+      end
+
+      it 'assigns @body_classes' do
+        subject
+        expect(assigns(:body_classes)).to eq 'app-body'
```

Comment:
These things which are just doing assertions against `assigns` are not super useful, and these methods were removed from rails a while ago (they are here only via the rails-controller-testing gem now), see rationale here: https://github.com/rails/rails/issues/18950

I think my preference would be that we eventually remove that gem, and improve either feature specs or request specs as is relevant in each case.